dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -4,7 +4,7 @@
     <CompanyInfoSheet :company-id="companyId" :show-single-data-request-button="true" />
     <div class="card-wrapper">
       <div class="card-grid">
-        <ClaimOwnershipPanel v-if="!isUserDataOwner && userId && isCompanyIdValid" :company-id="companyId" />
```

Comment:
In this component some changes will be necessary once you have merged in main

Reply:
yes